tests/TestCase/Authentication/Identifier/Resolver/ApiResolverTest.php

```diff
@@ -123,6 +123,7 @@ public function testFind(array $credentials, ?array $expected): void
         $token = $identity['tokens']['renew'];
         $identity = $resolver->find(compact('token'));
 
+        unset($expected['tokens'], $identity['tokens']);
```

Comment:
Rather than unsetting these values here, I'd remove the pieces of code that populate these values in the first place:

- lines 107-110 (`if isset($expected['tokens']) { ... }`)
- line 86 (`'tokens' => [],`)

Reply:
Right, done